WordPress/Classes/ViewRelated/Comments/EditCommentTableViewController.swift

```diff
@@ -100,12 +102,27 @@ class EditCommentTableViewController: UITableViewController {
     }
 
     override func tableView(_ tableView: UITableView, heightForFooterInSection section: Int) -> CGFloat {
-        // Make sure no SectionFooter is rendered
+        if needToShowRegisteredUserNotice(for: section) {
+            return UITableView.automaticDimension
+        }
+
         return CGFloat.leastNormalMagnitude
     }
 
+    override func tableView(_ tableView: UITableView, titleForFooterInSection section: Int) -> String? {
+        if needToShowRegisteredUserNotice(for: section) {
+            return NSLocalizedString("\nThis user is registered. Their name, web address, and email address cannot be edited.",
```

Comment:
This is minor, but I added a newline to separate the text from the bottom of the email address field a bit so the text stands out a bit more. If you disagree, I can remove it. I just thought it looked better.

Reply:
My only concern was whether our translators would preserve the `\n`, but I already see some localization strings that start with `\n` so it should be fine. 😄